src/nimblepkg/tools.nim

```diff
@@ -62,13 +62,8 @@ proc tryDoCmdEx*(cmd: string): string {.discardable.} =
     raise nimbleError(tryDoCmdExErrorMessage(cmd, output, exitCode))
   return output
 
-proc getNimBin*: string =
-  result = "nim"
-  if findExe("nim") != "": result = findExe("nim")
-  elif findExe("nimrod") != "": result = findExe("nimrod")
-
 proc getNimrodVersion*(options: Options): Version =
-  let vOutput = doCmdEx(getNimBin(options).quoteShell & " -v").output
+  let vOutput = doCmdEx(getNimBin(options) & " -v").output
```

Comment:
if you're gonna do that then rename `getNimBin` to `getNimBinQuoted`

Reply:
alternatively just revert this change, quoteShell is much better closest to doCmdExe to ensure we use it (plus other programs like choosenim might be using getNimBin and won't be expecting quoteShell)

Reply:
I will revert this refactoring and keep the `quoteShell` in the client.